packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-line-settings.tsx

```diff
@@ -69,6 +71,29 @@ const LockedLineSettings = (props: Props) => {
     // Check if the line has length 0.
     const isInvalid = kvector.equal(point1.coord, point2.coord);
 
+    function getPrepopulatedAriaLabel() {
+        let str = `${capitalizeKind} from (${point1.coord[0]}, ${point1.coord[1]}) to (${point2.coord[0]}, ${point2.coord[1]})`;
+
+        if (labels && labels.length > 0) {
+            str += " with label";
+            // Make it "with labels" instead of "with label" if there are
+            // multiple labels.
+            if (labels.length > 1) {
+                str += "s";
+            }
+
+            for (let i = 0; i < labels.length; i++) {
+                // Separate additional labels with commas.
+                if (i > 0) {
+                    str += ",";
+                }
+                str += ` ${labels[i].text}`;
+            }
+        }
+
+        return str;
+    }
```

Comment:
I notice that the points of a line can also have their own labels (at least, according to the `LockedPointType`). Do we want to include those as well?

Reply:
I thought about that, but I think I'm going to forego it because the more I thought about it, the more it sounded like something the content authors should manually write the aria label for. Especially since each point can hypothetically have multiple labels.